src/components/usa-modal/usa-modal.stories.js

```diff
@@ -0,0 +1,20 @@
+import "../../stylesheets/uswds.scss";
```

Comment:
Note: I do not see a close button on the modal. It looks like the img path needs to be updated.